services/admin/schema.sql

```diff
@@ -0,0 +1,5 @@
+create table adjudications (
+	id varchar(36) primary key,
```

Comment:
Seems like we should also be tracking the sheet id, no?
```suggestion
	id varchar(36) primary key,
	sheet_id text,
```